spec/Vehicle/Vehicle.vspec

```diff
@@ -184,16 +184,43 @@ Speed:
   description: Vehicle speed.
 
 TravelledDistance:
+  deprecation: V3.1 moved to Vehicle.TraveledDistance
   datatype: float
   type: sensor
   unit: km
-  description: Odometer reading, total distance travelled during the lifetime of the vehicle.
+  description: Odometer reading, total distance traveled during the lifetime of the vehicle.
 
-TripMeterReading:
+TraveledDistance:
   datatype: float
   type: sensor
   unit: km
-  description: Current trip meter reading.
+  description: Odometer reading, total distance traveled during the lifetime of the vehicle.
+
+TraveledDistanceSinceStart:
+  datatype: float
+  type: sensor
+  unit: km
+  description: Distance traveled since start of current trip.
+  comment: A new trip is considered to start when engine gets enabled (e.g. LowVoltageSystemState in ON or START mode).
+           A trip is considered to end when engine is no longer enabled.
+           The signal may however keep the value of the last trip until a new trip is started.
+
+TimeSinceStart:
+  deprecation: V3.1
+  datatype: uint32
+  type: sensor
+  unit: s
+  description: Time in seconds elapsed since start of current trip.
+  comment: A new trip is considered to start when engine gets enabled (e.g. LowVoltageSystemState in ON or START mode).
+           A trip is considered to end when engine is no longer enabled.
+           The signal may however keep the value of the last trip until a new trip is started.
+
+TripMeterReading:
```

Comment:
For traveled distance I see it relevant to have 3 values

- TraveledDistance - the value for the life time of the vehicle
- TraveledDistanceSinceStart - the value since start of current trip
- TripMeterReading - the value since manual reset (by driver)

Adapting description accordingly.